tests/foreman/ui/test_capsule.py

```diff
@@ -93,3 +93,349 @@ def test_positive_chost_selector(self):
 
         @Status: Manual
         """
+
+    @run_only_on('sat')
+    @stubbed
+    @tier1
+    def test_positive_capsule_version(self):
+        """Check Capsule Version in About Page.
+
+        @Feature: Capsule Management UX
```

Comment:
We use to use UI instead of UX. I think we should continue using UI and avoid introducing a new nomenclature.